### ISSUE TITLE
Confusing log messages PD_12#execute_prlctl

### ISSUE THREAD

**Issue by josefglatz** [Parallels/vagrant-parallels#311]
Everytime I vagrant up an vm I get the following lines.

So far the VM runs as it should be.

```
Bringing machine 'default' up with 'parallels' provider...
==> default: Checking if box 'bento/ubuntu-16.04' is up to date...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 0: shared
    default: Adapter 1: hostonly
==> default: Clearing any previously set network interfaces...
==> default: Running 'pre-boot' VM customizations...
/Users/jglatz/.vagrant.d/gems/2.4.2/gems/vagrant-parallels-1.7.7/lib/vagrant-parallels/action/customize.rb:29:in `block in call': VagrantPlugins::Parallels::Driver::Meta#execute_prlctl at /opt/vagrant/embedded/lib/ruby/2.4.0/forwardable.rb:157 forwarding to private method VagrantPlugins::Parallels::Driver::PD_12#execute_prlctl
/Users/jglatz/.vagrant.d/gems/2.4.2/gems/vagrant-parallels-1.7.7/lib/vagrant-parallels/action/customize.rb:29:in `block in call': VagrantPlugins::Parallels::Driver::Meta#execute_prlctl at /opt/vagrant/embedded/lib/ruby/2.4.0/forwardable.rb:157 forwarding to private method VagrantPlugins::Parallels::Driver::PD_12#execute_prlctl
/Users/jglatz/.vagrant.d/gems/2.4.2/gems/vagrant-parallels-1.7.7/lib/vagrant-parallels/action/customize.rb:29:in `block in call': VagrantPlugins::Parallels::Driver::Meta#execute_prlctl at /opt/vagrant/embedded/lib/ruby/2.4.0/forwardable.rb:157 forwarding to private method VagrantPlugins::Parallels::Driver::PD_12#execute_prlctl
/Users/jglatz/.vagrant.d/gems/2.4.2/gems/vagrant-parallels-1.7.7/lib/vagrant-parallels/action/customize.rb:29:in `block in call': VagrantPlugins::Parallels::Driver::Meta#execute_prlctl at /opt/vagrant/embedded/lib/ruby/2.4.0/forwardable.rb:157 forwarding to private method VagrantPlugins::Parallels::Driver::PD_12#execute_prlctl
==> default: Booting VM...
```

**Comment by romankulikov** [comment]
Confirm issue is reproduced on the latest version of vagrant.

**Comment by fractalzombie** [comment]
Have same logs:

> /Users/fractal/.vagrant.d/gems/2.4.2/gems/vagrant-parallels-1.7.7/lib/vagrant-parallels/action/box_register.rb:124:in `register_box': VagrantPlugins::Parallels::Driver::Meta#execute_prlctl at /opt/vagrant/embedded/lib/ruby/2.4.0/forwardable.rb:157 forwarding to private method VagrantPlugins::Parallels::Driver::PD_12#execute_prlctl

**Comment by legal90** [comment]
The issue was reproducible on Vagrant 2.0.1, where Ruby 2.4 was introduced. 
Fixed by #313
